PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
```

Comment:
The only value nesting both into a "Context" is for the developer. This way we declare both as being some Kind of Context and grouping both together. I am fine with both.